Source/SwiftLintFramework/Models/SwiftVersion.swift

```diff
@@ -23,4 +23,11 @@ enum SwiftVersion {
 
         fatalError("Unexpected Swift version")
     }()
+
+    var nameKey: String {
```

Comment:
But I could use "key.typename" directly there, as it's used inside the switch. What do you think would be better?